packages/nimble-components/build/generate-workers/source/matrix-renderer.ts

```diff
@@ -99,8 +120,20 @@ export class MatrixRenderer {
         }
     }
 
+    public setValues(valuesBuffer: Float64Array): void {
```

Comment:
Beyond the suggested new matrix tests, the only place I could see a reasonable unit test for is the logic for when `drawText` is executed within `drawWafer` in the `WorkerRenderer`. I'm not sure how easy that is to set up, and if it's too expensive/laborious, then I think the matrix tests will suffice.